### ISSUE TITLE
MediaPlayer: Disable then re-enable does not resume playing (workaround possible)

### ISSUE THREAD

**Issue by TheloTheGreat** [RenderHeads/UnityPlugin-AVProVideo#657]
When you disable a MediaPlayer that was playing, that pauses playback. When you re-enable it, that does not re-enable playback.

The C# code for MediaPlayer.OnDisable and OnEnable seems to intend for playback to resume in that situation, by setting _wasPlayingOnPause = true on disable:

```
void OnDisable()
{
	if (_controlInterface != null)
	{
		if (_controlInterface.IsPlaying())
		{
			_wasPlayingOnPause = true;
			Pause();
		}
	}
	StopRenderCoroutine();
}
```

However setting _wasPlayingOnPause = true there does nothing, since the Pause method immediately just resets _wasPlayingOnPause to false.

My workaround: If I modify MediaPlayer.OnDisable to set _wasPlayingOnPause = true; _after_ the Pause() call, then playback does resume when I re-enable the MediaPlayer. Maybe that's what you initially intended?

**Comment by kahnivore** [comment]
Which OS are you using?

**Comment by TheloTheGreat** [comment]
This is on both Windows 10 and Mac Big Sur, which doesn't matter since the issue is in the AVPro C# code.

**Comment by MorrisRH** [comment]
The fix will be in the next release

**Comment by TheloTheGreat** [comment]
Fix confirmed in 2.0.9, thanks.
